open-company/open-company-bot
# Block kit digest

Cards:
- https://trello.com/c/j7AkN7K6
- https://trello.com/c/Y3t8VhnW

Related PRs:
- [Lib](https://github.com/open-company/open-company-lib/pull/61)
- [Interaction](https://github.com/open-company/open-company-interaction/pull/32)
- [Infra](https://github.com/belucid/open-company-infrastructure/pull/103)
- [Email](https://github.com/open-company/open-company-email/pull/49)

To test:
- make sure you have interaction, bot and Slack router services running
- start ngrok on port 3009 (slack router)
- enable events subscription on local app on [Slack app settings page](https://api.slack.com/apps/A5SGHA79P/event-subscriptions?)
- add a user with the default avatar
- add some posts with that user
- send your Slack user a Slack digest
- [x] does it respect the new design? Good
- [x] do you NOT see the usage messages after the digest messages? Good
- now send a message in the Slack DM channel of the carrot local bot
- [x] do you get the usage message back? Good
- [x] if you want to do a more complete test uninstall the app from Slack and re-add it, check you get the welcome+usage message
- now send yourself an email digest
- [x] did you get it? Good
- [x] do you see the user avatars in it? Good
- upgrade lib version
- release lib version
- upgrade lib version in all the related services
- merge this PRs
- deploy


NB: the accessory image if any will be added in a second PR, this is only meant to work the transition from attachments to blocks
